### DISCUSSION TITLE
Freeze Frame customisation

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#21037] [Ideas] [answered]
Please add option that disable approach circle
![image](https://user-images.githubusercontent.com/59089206/199072056-035b46bf-c728-4a14-b1f4-9fbe66855c00.png)

Map - https://osu.ppy.sh/beatmapsets/1360563#osu/2815425 (up to 310 combo)

**Comment by mk56-spn** [comment, marked as answer]
you can use Hidden's "approach circle only" option for this exact purpose

![osu_2022-10-31_19-12-50](https://user-images.githubusercontent.com/74463310/199079920-2cebce1b-8e35-46ee-a61d-c5bf61ec93ca.png)
